odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -2,6 +2,21 @@
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+_TEXT_WARNINGS = {
+    ".xml": {
+        r'active.*widget="toggle_button"|widget="toggle_button".*active|'
+        r'widget="boolean_button".*active|active.*widget="boolean_button"':
+        'You can remove actions archive/unarchive'
+    },
+    ".py": {
+        r".*oldname":
+            "oldname is not supported yet. Create a migration script",
+        r"digits=[^, )]*get_precision":
+            'You can use string to qualify type of '
+            'precision without import nothing'
+    },
+}
+
```

Comment:
```suggestion
```

Please don't add text warnings on this PR.

I like the concept of text warnings, I think it's a good addition to the library.
But the actual warnings described here not so much, and having all in the same PR makes it more difficult to review.

You can create other PRs for the actual warnings implementation. Ideally, one PR per warning, so that it's easier to review.

Reply:
I remove all text warnings. In future PR this can be included